src/main/java/com/katalon/plugin/testrail/TestRailQueryingTestSuite.java

```diff
@@ -28,11 +30,14 @@ public List<TestCaseEntity> query(ProjectEntity project, TestSuiteEntity testSui
 
         String testRunId = TestRailHelper.parseId(s, "^R(\\d+)");
         PluginPreference preferences = getPluginStore();
-        TestRailConnector connector = new TestRailConnector(
-                preferences.getString(TestRailConstants.PREF_TESTRAIL_URL, ""),
-                preferences.getString(TestRailConstants.PREF_TESTRAIL_USERNAME, ""),
-                preferences.getString(TestRailConstants.PREF_TESTRAIL_PASSWORD, "")
-        );
+        TestRailConnector connector = null;
+        try {
+            connector = new TestRailConnector(preferences.getString(TestRailConstants.PREF_TESTRAIL_URL, ""),
+                    preferences.getString(TestRailConstants.PREF_TESTRAIL_USERNAME, ""),
+                    preferences.getString(TestRailConstants.PREF_TESTRAIL_PASSWORD, "", true));
+        } catch (Exception e) {
+            e.printStackTrace(System.out);
```

Comment:
- Any action here?
- Should try catch all possible Exception, not set Exception as a general one.